### PR TITLE
12.0 add stock picking unreserve line

### PR DESCRIPTION
**Pull request by MPoch-PlanetaTIC**
Permits to unreserve stock.move

### CODE REVIEW

**Review by ypapouin — CHANGES_REQUESTED** [review]
Don't you think that a specific security group could be added for this ?

File: stock_picking_unreserve_line/readme/CONSTRIBUTRORS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* Marc Poch <mpoch@planetatic.com>
```
Comment (StefanRijnhart) [inline stock_picking_unreserve_line/readme/CONSTRIBUTRORS.rst:1]: Typo in file name `CONSTRIBUTRORS.rst `

File: stock_picking_unreserve_line/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Permits to unreserve stock.move
```
Comment (StefanRijnhart) [inline stock_picking_unreserve_line/readme/DESCRIPTION.rst:1]: Can you be a bit more verbose? What can the user expect after the installation of the module?

**Comment by StefanRijnhart** [comment]
@ypapouin in core Odoo, there is no security group on the picking's Unreserve button, so adding one here would not be necessary I think https://github.com/odoo/odoo/blob/12.0/addons/stock/views/stock_picking_views.xml#L222

File: stock_picking_unreserve_line/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+{
+    'name': 'Stock Picking Unreserve Line',
+    'summary': 'Permits to unreserve lines of stock picking',
+    'version': '12.0.1.0.0',
+    'category': 'Stock',
+    'author': 'PlanetaTIC'
```
Comment (StefanRijnhart) [inline stock_picking_unreserve_line/__manifest__.py:10]: Missing comma after PlanetaTIC breaking the linting

**Review by StefanRijnhart — CHANGES_REQUESTED** [review]
Please add a simple test

**Pull request closed without merging**